### PR TITLE
Added few functions used in rules of RUBI

### PR DESCRIPTION
**Pull request by ashishkg0022**
I was trying to run other Integration tests of RUBI.There were functions that were not defined.
1. `DerivativeDivides`
2. `LogIntegral`
Is the implementation of `LogIntegral` fine ? I am not sure about its right implementation. 
I will add more functions similar to LogIntegral, If its implementation is right.

### CODE REVIEW

**Comment by ashishkg0022** [comment]
ping @Upabjojr

File: sympy/integrals/rubi/utility_function.py [hunk 1]
```diff
@@ -5395,6 +5395,36 @@ def Divides(y, u, x):
...
+        if PolynomialQ(y, x):
+            return PolynomialQ(u, x) and Exponent(u, x)==Exponent(y, x)-1
+        else:
+            return EasyDQ(y, x)
+
+    if is_match(y, pattern0):
+        return False
+
+    elif f1(y, u, x):
+        v = D(y ,x)
```
Comment (Upabjojr) [inline sympy/integrals/rubi/utility_function.py]: in Rubi 4.14 there is also `ReplaceAll[D[y,x],Sinc[z_]->Sin[z]/z]]`. Are you using a different version?

**Comment by ashishkg0022** [comment]
Is it needed ? Cant we represent `sin(z)/z` as it is ? We have no where used `Sinc` in Rubi .

**Comment by Upabjojr** [comment]
> Is it needed ?

No, it isn't.

**Pull request merged**